app/components/widgets/settings/radio_setting.js

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {Text, TouchableOpacity, View} from 'react-native';
+import CheckMark from 'app/components/checkmark';
+import {paddingHorizontal as padding} from 'app/components/safe_area_view/iphone_x_spacing';
+import FormattedText from 'app/components/formatted_text';
+import {makeStyleSheetFromTheme, changeOpacity} from 'app/utils/theme';
+
+export default class RadioSetting extends PureComponent {
+    static propTypes = {
+        id: PropTypes.string.isRequired,
+        label: PropTypes.node.isRequired,
+        values: PropTypes.array.isRequired,
+        value: PropTypes.string.isRequired,
```

Comment:
is value a prop, or should it be part of a local state?

On the other hand, you'd need a default value which definitely is a props, maybe we should split it into two?